src/class6Library/Author.java

```diff
@@ -0,0 +1,17 @@
+package class6Library;
+
+public class Author {
```

Comment:
equals method is missing )) good to have,
it will be great to have more fields in the Author class)